app/src/main/java/org/listenbrainz/android/util/Constants.kt

```diff
@@ -11,7 +11,7 @@ object Constants {
     const val FEEDBACK_EMAIL = "support@metabrainz.org"
     const val FEEDBACK_SUBJECT = "[LBAndroid] Feedback"
     const val SPOTIFY_REDIRECT_URI = "org.listenbrainz.android://callback"
-    const val LISTENBRAINZ_API_BASE_URL = "https://api.listenbrainz.org/1/"
+    const val LISTENBRAINZ_API_BASE_URL = "https://beta-api.listenbrainz.org/1/"
```

Comment:
You can't be doing this. This changes things across the repo. Revert this please. For testing YIM stuff, you should have created a BETA_LISTENBRAINZ_API_BASE_URL and used it.

Reply:
Yup I am sorry, I shudn't have commited it